dbt/config.py

```diff
@@ -5,9 +5,6 @@
 
 
 def read_config(profiles_dir=None):
-    if profiles_dir is None:
-        profiles_dir = default_profiles_dir
-
     path = os.path.join(profiles_dir, 'profiles.yml')
```

Comment:
this is a good call. since `read_config` is part of the private API of this namespace, and calls have to go through `send_anonymous_usage_stats`, i just removed the default `None` value.